tests/bdd/features/helm_chart/test_install.py

```diff
@@ -0,0 +1,251 @@
+"""Install Mayastor feature tests."""
+
+from common.k8s import (
+    deployment_is_available,
+    daemon_set_is_available,
+    is_k8s_control_plane_node,
+    stateful_set_is_available,
+)
+from common.utils import retry_predicate
+from kubernetes import (
+    client as kube_client,
+    config as kube_config,
+)
+import os
+import pytest
+from pytest_bdd import (
+    given,
+    scenario,
+    then,
+    when,
+)
+from subprocess import run
+
+
+@scenario('install.feature', 'Mayastor helm chart install')
+def test_mayastor_helm_chart_install():
+    """Mayastor helm chart install."""
+
+
+@given('a kubernetes namespace to install the Mayastor helm chart')
+def a_kubernetes_namespace_to_install_the_mayastor_helm_chart():
+    """a kubernetes namespace to install the Mayastor helm chart."""
+    kube_config.load_kube_config()
```

Comment:
Should there create_namespace, label et all be part of a common thing? Seems like we'd be doing this a lot?

Reply:
Resolving this conversation, because this test is no longer a part of this PR.